aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -310,6 +310,22 @@ public void testRenameTable() {
     assertThat(renamedTable.currentSnapshot()).isEqualTo(table.currentSnapshot());
   }
 
+  @Test
+  public void testCreateTableAfterRename() {
```

Comment:
I would maybe rename this to `createTableInUniqueLocation()` here and in all the other tests. This makes it easier to "guess" what the test is trying to achieve

Reply:
Make sense, done